src/algo/mod.rs

```diff
@@ -59,6 +59,9 @@ pub use matching::{greedy_matching, maximum_matching, Matching};
 pub use maximal_cliques::maximal_cliques;
 pub use min_spanning_tree::{min_spanning_tree, min_spanning_tree_prim};
 pub use page_rank::page_rank;
+#[allow(deprecated)]
+pub use scc::scc;
+pub use scc::{kosaraju_scc, tarjan_scc, TarjanScc};
```

Comment:
```suggestion
pub use scc::{
    kosaraju_scc::kosaraju_scc,
    tarjan_scc::{tarjan_scc, TarjanScc},
};
```

To avoid making noise with modules in re-exports.